### PR TITLE
Feat: Implement Account Details Page

### PR DESCRIPTION
**Pull request by evilpeach**

### CODE REVIEW

File: src/config/chain/osmosis.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const OSMOSIS_CHAIN_CONFIGS: ChainConfigs = {
     prettyName: "Osmosis Testnet",
     lcd: "https://lcd.osmotest5.osmosis.zone",
     rpc: "https://osmosis-testnet-rpc.polkachu.com:443",
-    indexer: "https://osmo-test-5-graphql.alleslabs.dev/v1/graphql",
+    indexer: "",
```
Comment (songwongtp) [inline src/config/chain/osmosis.ts:61]: revert

File: src/lib/pages/account-details/data.ts [hunk 2]
```diff
@@ -34,13 +36,29 @@ interface AccountDetailsTableCounts {
...
+  } = useAccountTableCounts(address, { enabled: isFullTier });
   const { data: balances, isLoading: isBalancesLoading } = useBalances(address);
 
+  if (!isFullTier) {
+    return {
+      tableCounts: {
+        codesCount: undefined,
+        contractsAdminCount: undefined,
+        contractsCount: undefined,
+        txsCount: undefined,
```
Comment (songwongtp) [inline src/lib/pages/account-details/data.ts:53]: can we use instatiated contract by address LCD hook to get the count?
also for txs does `pagination.count_total` work? or too slow

Comment (evilpeach) [reply src/lib/pages/account-details/data.ts:53]: Most of the chain doesn't support count_total on instantiated contracts by address as follows [link](https://terra-testnet-api.polkachu.com/cosmwasm/wasm/v1/contracts/creator/terra1vgm7t8q43sfg0xq9693ah0xvgxv0qnkm9xtr54?pagination.count_total=true)

File: CHANGELOG.md [hunk 3]
```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Features
 
+- [#970](https://github.com/alleslabs/celatone-frontend/pull/970) Support account details lite version (basic + wasm)
```
Comment (songwongtp) [inline CHANGELOG.md:42]: Remove (basic+wasm)

File: src/lib/services/account/lcd.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import axios from "axios";
+
+import type { AccountData } from "../types";
+import type { BechAddr } from "lib/types";
+
+export const getAccountDataLcd = async (
```
Comment (songwongtp) [inline src/lib/services/account/lcd.ts:7]: will this be in another PR?

**Review by songwongtp — APPROVED** [review]

File: src/lib/services/account/lcd.ts (hunk 4 above)
Comment (evilpeach) [reply src/lib/services/account/lcd.ts:7]: Yes
